cmd/kubeadm/app/cmd/options/generic.go

```diff
@@ -80,3 +82,10 @@ func AddKubernetesVersionFlag(fs *pflag.FlagSet, kubernetesVersion *string) {
 		`Choose a specific Kubernetes version for the control plane.`,
 	)
 }
+
+// AddOutputFlag adds the --output/-o flag to the given flagset
+func AddOutputFlag(fs *pflag.FlagSet, outputFormat *string) {
+	fs.StringVarP(
+		outputFormat, "output", "o", output.TEXTOutput,
+		fmt.Sprintf("Output format; available options are: %s, %s, %s and Go template", output.TEXTOutput, output.YAMLOutput, output.JSONOutput))
```

Comment:
We might have a slice with all supported output formats for this purpose and for validation too.